README.md

```diff
@@ -65,6 +65,36 @@ As of v2.0, the storage solution for making this data available alongside the no
 
 Data relating to the **CSSP 20CRDS** tutorials is held online in an Azure Blob Storage Service. To access this data user will need a valid shared access signature (SAS) token.  The data is in [Zarr](https://zarr.readthedocs.io/en/stable/) format and the total volume is ~2TB. The data is in hourly, 3 hourly, 6 hourly, daily and monthly frequencies stored seperatrely under the `metoffice-20cr-ds` container on MS-Azure. Monthly data only is also via [Zenodo](https://zenodo.org/record/2558135).
 
+
+# create s3 bucket
+
+* goto s3 service and press "create bucket"
+* name the bucket 
+* set region to EU (London) eu-west-2
+* add tags:
+  * Name: [name of bucket or any unique name]
+  * ServiceOwner: [your-name]
+  * ServiceCode: PABCLT
+  * Tenable: FA
+* click on "create bucket" 
+
+## Loading data on s3 bucket from VDI (using boto3)
+
+to rupload the data on S3 use: /aws-scripts/s3_data_upload.py
+
+## Key configurations
+
+The above script run only when config files contains latest keys. In order to update the keys: 
+
+* go to AB climate training dev --> Administrator access --> command line or programmatic access
+* Copy keys in "Option 1: Set AWS environment variables"
+* In VDI, paste (/replace existing) these keys in ~/.aws/config
+* Copy keys in "Option 2: Add a profile to your AWS credentials file"
+* In VDI, Paste the keys in credentials file: ~/.aws/credentials (remove the first copied line, looks somethings like: [198477955030_AdministratorAccess])
+
```

Comment:
shouldn't these have the permissions set to be only allow the owner to see them?

When I try and run the upload script I get the error

configparser.MissingSectionHeaderError: File contains no section headers.
file: '/home/h03/fris/.aws/config', line: 1
'export AWS_ACCESS_KEY_ID="ASIAS4NRVH7LNAZ5ZI6V"\n'

Reply:
if I just pasted the environment variables into the shell I ran the Python script from it does work.

Reply:
exporting the keys in config and credential file is advised mainly because if someone restarts the machine, the keys will work. but issue is key updates every next days.

Reply:
> exporting the keys in config and credential file is advised mainly because if someone restarts the machine, the keys will work. but issue is key updates every next days.

I understand that, but when I follow the instructions in your document, I can't get it to work